src/main/java/org/arl/fjage/shell/ConsoleShell.java

```diff
@@ -123,25 +157,15 @@ public void init(ScriptEngine engine) {
     if (scriptEngine == null)
       console = LineReaderBuilder.builder().terminal(term).option(LineReader.Option.AUTO_FRESH_LINE, true).build();
     else {
-      Parser parser = new Parser() {
-        @Override
-        public CompletingParsedLine parse(String s, int cursor) {
-          if (!scriptEngine.isComplete(s)) throw new EOFError(-1, -1, "");
-          if (s.contains("\n") && cursor < s.length()) throw new EOFError(-1, -1, "");
-          return null;
-        }
-        @Override
-        public CompletingParsedLine parse(String s, int cursor, Parser.ParseContext context) {
-          return parse(s, cursor);
-        }
-        @Override
-        public boolean isEscapeChar(char ch) {
-          return false;
-        }
-      };
-      console = LineReaderBuilder.builder().parser(parser).terminal(term).build();
-      console.setVariable(LineReader.DISABLE_COMPLETION, true);
-      console.setOpt(LineReader.Option.ERASE_LINE_ON_FINISH);
```

Comment:
Was this removal done on purpose? It changes the shell behaviour quite significantly. 

On master, when entering `[<enter>` I get
```
> [
- 
```

On this branch, I get
```
> [
[: ']' expected
```
and then the fjage shell gets closed!

Reply:
The custom parser was causing NPE in the autosuggestion widget. The `setVariable` and `setOpt` changes were from the TabCompleter tests I did. I can revert them back.


Will also fix this issue

Reply:
OK. I tried this. I can't seem to reproduce this. Anyway I added back the `setVariable` and `setOpt`.

https://asciinema.org/a/HbablmoZ3PerEnsSCw0Tfp8SF